pkg/kyma/cmd/provision/minikube/hosts_darwin.go

```diff
@@ -11,18 +16,65 @@ const hostsFile = "/etc/hosts"
 
 const defaultVMDriver = vmDriverHyperkit
 
+func isWithSudo() bool {
```

Comment:
Although this is correct, to me it looks more readable as:
```
func isWithSudo() bool {
   return os.Getenv("SUDO_UID") != ""
}
```
It avoids having to go through the branches of the `if-else` and putting the pieces together in my head.

Reply:
Fixed